config/application.rb

```diff
@@ -48,7 +48,6 @@ class Application < Rails::Application
     # Enable the asset pipeline
     config.assets.enabled = true
 
-    # TODO: Move to asset pipeline enabled by moving assets from vmdb/public to vmdb/app/assets
```

Comment:
@romanblanco Does this mean we're fully using the asset pipeline? I still see `public/javascripts/timeline/api`.  I'd rather clarify/correct this comment until we move everything to the asset pipeline and remove the comment and the asset_path setting.